src/server2.py

```diff
@@ -0,0 +1,40 @@
+# coding=utf-8
+from server import response_ok, response_error, parse_request, resolve_uri
+
+
+def echo(socket, address):
+    """Run Server."""
+
+    buffer_length = 8
+    message_complete = False
+    return_message = ''
+    while not message_complete:
+        part = socket.recv(buffer_length)
+        if len(part) < buffer_length:
+            message_complete = True
+        return_message += part.decode('utf-8')
+    print('Return Message: ' + return_message)
+    try:
+        uri = parse_request(return_message)
+        content = resolve_uri(uri)
```

Comment:
Why not this instead:

```python
content, content_type = resolve_uri(uri)
```
You know you'll be getting back two values, and that the first is the content and the second the appropriate value for the Content-Type header.  Why not just use multiple assignment instead of indexing the tuple in the lines below this?